tests/Erdcsharp.Tests/Domain/GasLimitTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Threading.Tasks;
+using Erdcsharp.Domain;
+using Erdcsharp.Provider;
+using Erdcsharp.Provider.Dtos;
+using Moq;
+using NUnit.Framework;
+
+namespace Erdcsharp.UnitTests.Domain
+{
+    [TestFixture(Category = "UnitTests")]
+    public class GasLimitTests
+    {
+        private IElrondProvider _elrondProvider;
+
+        [SetUp]
+        public void Setup()
+        {
+            var mock = new Mock<IElrondProvider>();
+            mock.Setup(s => s.GetNetworkConfig()).ReturnsAsync(new ConfigDataDto
+            {
+                Config = new ConfigDto
+                {
+                    erd_min_gas_limit     = 50000,
```

Comment:
Now I see - can we define pretty properties on this DTO, but decorate them with some JSON metadata / attributes?